agent/api/task/task_windows.go

```diff
@@ -105,6 +109,13 @@ func (task *Task) platformHostConfigOverride(hostConfig *dockercontainer.HostCon
 		hostConfig.MemoryReservation = 0
 	}
 
+	// On Windows, bridge mode equivalent for Linux is "default" or "nat" network.
+	// We need to perform explicit conversion for the same.
```

Comment:
A mental note on the comment above and how to think about these concepts:
The "bridge" networkMode here is ECS' network mode, not docker's. The fact that docker also has a networking mode called "bridge" should be treated as coincidental. Today ECS' bridge is implemented with docker's bridge under the hood, but that can change in the future.

Therefore, instead of saying "bridge mode equivalent is 'nat'", we should say "On Windows, ECS' bridge mode is implemented with "nat" mode."